apps/meteor/server/models/raw/Users.js

```diff
@@ -1657,11 +1657,11 @@ export class UsersRaw extends BaseRaw {
 		return this.findOne(query);
 	}
 
-	findOneOnlineAgentById(_id, isLivechatEnabledWhenAgentIdle) {
+	findOneOnlineAgentById(_id, options, isLivechatEnabledWhenAgentIdle) {
```

Comment:
Would it not be better to have options as the last parameter to follow the other cases?

Reply:
You're right, not sure why I've done that